README.md

```diff
@@ -117,6 +117,71 @@ python -m build
 # 📈 Training
 
 The torch-ort library accelerates training of large transformer PyTorch models to reduce the training time and GPU cost with a few lines of code change. It is built on top of highly successful and proven technologies of ONNX Runtime and ONNX format and includes the ONNX Runtime Optimizer and Data Sampler.
+## Add ONNX Runtime for PyTorch to your PyTorch training script
+
+```python
+from torch_ort import ORTModule
+model = ORTModule(model)
+# PyTorch training script follows
+```
+
+## Usage of FusedAdam and FP16 Optimizer (Optional)
```

Comment:
Why optional?

Reply:
like its not required to be used in this example, only if you want to use fp16 you can add that line.

Reply:
I don't think we need to provide the optional keyword. I think with some details we should be ok.